### PR TITLE
Figure for the "D in Eurofer" case

### PR DESCRIPTION
**Pull request by KulaginVladimir**
Adds figure for the final case

### CODE REVIEW

File: D_EUROFER/D_EUROFER.ipynb
Comment (RemDelaporteMathurin) [inline D_EUROFER/D_EUROFER.ipynb:1]: I would make the line in the legend red and not black

Comment (KulaginVladimir) [reply D_EUROFER/D_EUROFER.ipynb:1]: What if we also change the color to orange? This orange is from the colour palette we used in other cases
image

Comment (RemDelaporteMathurin) [reply D_EUROFER/D_EUROFER.ipynb:1]: Not bad!

Comment (KulaginVladimir) [reply D_EUROFER/D_EUROFER.ipynb:1]: Well, I can't decide. Red+grey looks better, but then we have 7+ colours <- not the best option. In general, you may choose if smth looks fine

Comment (KulaginVladimir) [reply D_EUROFER/D_EUROFER.ipynb:1]: 7+ colours in the whole paper: 5 are from the colourmap in figs. 2-3 and grey/red from figs. 5-6

**Pull request merged**